src/molecules/Modal/modal.module.scss

```diff
@@ -0,0 +1,87 @@
+@import 'assets/stylesheets/global';
+
+$modal-standard-max-width:                  100%;
+$modal-wide-max-width:                      100%;
+$modal-image-max-width:                     rem-calc(300px);
+$modal-border-color:                        color('neutral-5');
+$modal-header-background-color:             color('neutral-6');
+$modal-header-color:                        color('neutral-2');
+$modal-body-background-color:               color('neutral-8');
+$modal-body-color:                          color('neutral-2');
+$modal-body-subtitle-color:                 color('neutral-3');
+$modal-tab-border-color:                    color('neutral-5');
+$modal-tab-background-color:                color('neutral-7');
+$modal-tab-active-background-color:         color('neutral-8');
+$modal-tab-active-color:                    color('neutral-1');
+$robust-card-mobile-top-panel:              color('neutral-7');
+$robust-card-mobile-top-panel-border-color: color('neutral-5');
+$robust-card-mobile-top-panel-name-color:   color('neutral-3');
```

Comment:
@jmcolella It doesn't look like any of these variables are used in this file? Please remove them if not. Also, I know this was a pattern from the old library, but I don't think we need to list everything as a variable up top like this. We really only need to use variables if two or more properties are sharing the same values and only when it makes sense, imo.